src/shared/auto-update-cache.ts

```diff
@@ -0,0 +1,58 @@
+import { TimeString } from '@types';
```

Comment:
Once the cache is built, don't forget to export it (and it's config) in `src/shared/index.ts`